fe/index.php

```diff
@@ -1,124 +1,32 @@
+<?php
```

Comment:
this file (index.php) should be at the root of the project rather than in the fe directory

Reply:
Cheers for saving me on this one!